man/adoc/bpftrace.adoc

```diff
@@ -210,12 +210,16 @@ Increasing the value will consume more memory, increase startup times and can in
 
 === BPFTRACE_CACHE_USER_SYMBOLS
 
-Default: 0 if ASLR is enabled on system and `-c` option is not given; otherwise 1
+Default: PER_PROGRAM if ASLR disabled, PER_PID if ASLR enabled and -p option used; NONE otherwise.
```

Comment:
This doesn't match the implemented behaviour, although the way it's described here sounds good to me

Reply:
I don't see any difference between the behavior described here and the behavior implemented in `src/main.cpp`. Anyway, I'll change both to reflect the suggestion to default to PER_PID.